Cargo.toml

```diff
@@ -39,7 +39,10 @@ mio = { version = "0.7.13", features = ["os-ext", "os-poll"] }
 chrono = { version="0.4", features = ["serde"] }
 once_cell = "1.6.0"
 futures = { version = "0.3", features = ["thread-pool"] }
-oci-spec = { git="https://github.com/utam0k/oci-spec-rs/", tag = "v0.4.0-with-bugfix"}
+# TODO: Fetch from crate.io instead of git when next release oci-spec-rs
```

Comment:
Can you file an issue so we don't forget? Can you also file an issue in `oci-spec-rs` to ask for a minor release?

Reply:
OK. The following was submitted.

https://github.com/containers/youki/issues/329
https://github.com/containers/oci-spec-rs/issues/72